patterns/sidebar-writer.php

```diff
@@ -25,7 +25,7 @@
 <!-- /wp:separator -->
 
 <!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
-<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Derniers articles', 'à propos du titre des Derniers articles', 'twentytwentyfour' ); ?></h2>
+<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html_x( 'Derniers articles', 'à propos du titre des Derniers articles', 'twentytwentyfour' ); ?></h2>
```

Comment:
This text needs to be in English.

Reply:
@carolinan 

its already in English in latest commit. can you please review latest commit ?
Latest Commit : https://github.com/WordPress/twentytwentyfour/pull/195/commits/851682d7b35d24a153dd903587970991b973f6e6

Reply:
There is still several places that do not have English text.

Reply:
@carolinan 

Yes, its because @MaggieCabrera has suggested do not change text 
https://github.com/WordPress/twentytwentyfour/pull/195#discussion_r1315702220

Reply:
the theme has 3 "variations" business (default), writer and portfolio. Each of them has the content in a different language. This is intentional and I asked @kafleg and it's something that it's ok to have on a theme in the directory, so I don't feel like there's a reason to change the content. Especially not using google translate to do so.

Reply:
Doing that would make it very difficult to translate the theme, you would need 3 times the number of translators for each supported language, or translators that speak four languages.
I disagree strongly with this. I do understand the decision is not mine to make, and I apologize for creating extra work for everyone.  I would like to ask you to please discuss this with the polyglots team, and that you keep discussions public.
There is no way for me to know that this has been discussed already.

Reply:
No, there hasn't been any discussion. @beafialho, do you think we could change this?

Reply:
WordPress Core **requires** all default text to be in US-English. 
en_us is the default language of all code in WP, and if translation files are missing, all text is considered to be the default language. Adding text in other languages will be _more difficult_ instead of easier.